src/layouts/Main/components/Footer/Footer.js

```diff
@@ -81,13 +81,13 @@ const Footer = ({customRouting,colorInvert = false }) => {
               <Grid item xs={3}></Grid>
             </Grid>
           </Grid>
-          <Grid item xs={8}>
+          <Grid item xs={8} paddingLeft= {'0px !important'}>
```

Comment:
Please use mui spacing  here: https://mui.com/system/spacing/

Reply:
@ardeay I need to do that to override the current mui style

Reply:
check out that doc and try to do it using mui